site/content/docs/08-link-options.md

```diff
@@ -19,4 +18,16 @@ We can mitigate that by *prefetching* the data. Adding a `rel=prefetch` attribut
 
 > `rel=prefetch` is a Sapper idiom, not a standard attribute for `<a>` elements
 
-<!-- TODO add a function to prefetch programmatically -->
\ No newline at end of file
+<!-- TODO add a function to prefetch programmatically -->
+
+### rel=external
+
+The sapper router listens for clicks on `<a>` elements and bypasses the normal browser navigation for relative paths. We sometimes need links that are targetted at routes that are not part of the sapper app and hence should be reached by a normal browser navigation.
+
+Adding a `rel=external` attribute to a link...
+
+```html
+<a rel=prefetch href='path'>Path</a>
```

Comment:
Should this be `external` instead of `prefetch`?

Reply:
Yes, thank you.